packages/collection-config-store/tests/test-collection-config-store.js

```diff
@@ -0,0 +1,207 @@
+'use strict';
```

Comment:
Nit/git thought/rando stream of consciousness: 

I wonder if we should  have git moved these, then added a new copy to the soon-to-be-deprecated tests to preserve line/commit history and/or for ease of review?        

This file *does* differ from ../cumulus/packages/common/tests/collection-config-store.js, so manual diffing is a bit painful.

Reply:
Updates look good, moved from ../cumulus/packages/common/tests/collection-config-store.js